### ISSUE TITLE
[Question] Does this library unoficially support JSON comments?

### ISSUE THREAD

**Issue by ads103** [bblanchon/ArduinoJson#88]
I know that JSON doesn't officially support comments (which I think is pretty dumb), but some libraries DO have unofficial support for comments that resemble C multi-line comments. Does this library include such unofficial support? 
For example, I'd like to do something like this:
```
{
"Phone Numbers":[[9990001234, 5], [9990001235, 2], [9990001236, 1]] /* You may type your phone numbers here in format [number, priority]*/
}
```

**Comment by bblanchon** [comment]
It does not support comments. Sorry.

**Comment by bblanchon** [comment]
Feature available in [version 5.0 beta 4](https://github.com/bblanchon/ArduinoJson/releases/tag/v5.0-beta-4)
